### ISSUE TITLE
Integrate features from my Mineshot fork

### ISSUE THREAD

**Issue by Fuzss** [pascallj/mineshot-revived-fabric#15]
Hey there, nice to see you continue Barracuda's work.

I used Mineshot quite a lot back in the 1.12 and also went ahead and added my own features (mainly a GUI for finer control).
Maybe you'd be interested in cherry-picking parts of that? Especially the GUI was very hand back in the day.

I unfortunately don't have the time to make a PR, my code is on 1.12 after all and is also very much rookie code.

Here is my code, feel free to take whatever you want: https://github.com/Fuzss/mineshot

**Comment by pascallj** [comment]
Hi, thank you for your suggestion. There are definitely some cool features in your fork! Don't worry about rookie code, mine is not that great either :smile:.

Unfortunately the codebase has changed a lot since 1.12. I also moved to Fabric instead of Forge because I much prefer that. So it won't be just cherry-picking features, but more writing the code from scratch based on your concepts. Unfortunately I think my time is better spent on other projects (which I have too many of) unless there is a high demand for these features. My main focus for this project is keeping it alive and compatible with new Minecraft versions. But maybe in the future. I will definitely keep it in mind.

Do you mind giving some examples what features (and in what context) are particularly useful? I can see benefits in presets and a nice GUI for setting the angles (which we already have now in the settings) and such, but I can't see why you need that fine control over the angles with previews for example.

**Comment by Fuzss** [comment]
I implemented the GUI as it was needed for the Minecraft Wiki to take those isometric screenshot with the same settings each time and the GUI made that possible. Overall the precision at which it allows for setting values was just very handy for that. But yeah, that was the main intention...

**Comment by pascallj** [comment]
In that case it might be nice to know that the default settings are the correct ones for the Minecraft Wiki. So if you exit and enter isometric mode again, the default setting (or the ones you entered in the settings of you changed them) will be applied. Not sure if it was this way before, but it is now 😄

**Comment by Fuzss** [comment]
Yeah that part was the same before. It's more about the zoom level, getting a consistent zoom level was important for things like cycling images on the wiki to ensure the scale would match. Is there a way to manually set a zoom level now?

**Comment by pascallj** [comment]
No there is not, good point. If anything, that would be the first thing to add to the settings the same way as the x and y values. That is a couple of minutes work. 

You would have to position the player always in the exact same spot for that to work though right?

**Comment by Fuzss** [comment]
No, with the isometric perspective that’s not necessary since there is no distortion to the field of view.

**Comment by pascallj** [comment]
True, I meant for the alignment. If you want to cycle images on the wiki, the alignment has to be the same as well. But that's not much work in post processing I guess.
